src/engraving/libmscore/clef.cpp

```diff
@@ -25,6 +25,8 @@
  Implementation of classes Clef (partial) and ClefList (complete).
 */
 
+#include <QCoreApplication>
```

Comment:
curious, why QCoreApplication is needed here

Reply:
in fact, in most places at the moment, we need `QCoreApplication`, because of translation.
But I did not waste time on this and just replaced `QApplication` with `QGuiApplication` with auto-replace. (because `QApplication` is `QWidget`)
Because in any case, we have to use our translation function and all this will be removed.

Reply:
OK, let's create a separate task then

Reply:
see #8500